### DISCUSSION TITLE
Can I adjust the Canvas Size or page margins in Xournalpp (or Xournal)??

### DISCUSSION THREAD

**Discussion by goemonburo** [xournalpp/xournalpp#4812] [General] [answered]
Very sorry if this is a dumb question, I've searched the manual, combed through the options, and can't seem to find a way to add additional margins to my pdf files so that the comments and such can be viewed NOT atop the existing text or diagrams, but to the side.  Is there any way to do this?  

I'm running Ubuntu 20.04LTS.  Ideally I want to use my Wacom to circle a specific thing and draw a line to the outside margin, then shift to Type and add comments and notes.  Like "Make this larger" or "Needs to be in #fa12bc" or such.  

I have hundreds of existing PDFs that need this, so I'm hoping I don't have to go into each one and create a wider version with enough margin to do what I need.  

Thank you so much for any suggestions or advice.

**Comment by netlimpopo** [comment, marked as answer]
Right now one can only add some blank pages inbetween 2 pdf pages. The blank pages are best to be of a lean format. Then arranging the view with the toolbar leads to them being next to each other. This is a rather cumbersome workaround, but as far as I know that's everything that's possible so far. Sorry.

**Comment by goemonburo** [comment]
Thank you for the info.  I appreciate it!
